### PR TITLE
Fixes two bugs with hivemind

### PR DESCRIPTION
**Pull request by Penwin0**
## About The Pull Request
Assert dominance can be acquired properly now, awakened vessels will now properly get affected by One Mind

## Why It's Good For The Game
bugs bad

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Screenshot_1319

</details>

## Changelog
:cl:
fix: Hiveminds can properly get Assert Dominance now
fix: Awakened vessels are properly affected by One Mind
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]
epic dominance fail (also dominance should be stronger I think, but you've seen my comments I put in the thread in #development I assume)

File: code/modules/antagonists/hivemind/hivemind.dm [hunk 1]
```diff
@@ -83,7 +83,8 @@
 			break
 		if(lead)
 			unlocked_dominance = TRUE
-			owner.AddSpell(/obj/effect/proc_holder/spell/self/hive_dominance)
+			var/obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell = new
+			owner.AddSpell(dominance_spell)
```
Comment (Archanial) [inline code/modules/antagonists/hivemind/hivemind.dm]:
```suggestion
			owner.AddSpell(new obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell)
```

Comment (Penwin0) [inline code/modules/antagonists/hivemind/hivemind.dm]:
```suggestion
			owner.AddSpell(/new obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell)
```

File: code/modules/antagonists/hivemind/hivemind.dm [hunk 2]
```diff
@@ -83,7 +83,8 @@
 			break
 		if(lead)
 			unlocked_dominance = TRUE
-			owner.AddSpell(/obj/effect/proc_holder/spell/self/hive_dominance)
+			var/obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell = new
```
Comment (Penwin0) [inline code/modules/antagonists/hivemind/hivemind.dm]:
```suggestion
```

File: code/modules/antagonists/hivemind/hivemind.dm [hunk 3]
```diff
@@ -83,7 +83,7 @@
 			break
 		if(lead)
 			unlocked_dominance = TRUE
-			owner.AddSpell(/obj/effect/proc_holder/spell/self/hive_dominance)
+			owner.AddSpell(/new obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell)
```
Comment (Penwin0) [inline code/modules/antagonists/hivemind/hivemind.dm]:
```suggestion
			owner.AddSpell(new /obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell)
```

**Review by Archanial — APPROVED** [review]

File: code/modules/antagonists/hivemind/hivemind.dm [hunk 4]
```diff
@@ -83,7 +83,7 @@
 			break
 		if(lead)
 			unlocked_dominance = TRUE
-			owner.AddSpell(/obj/effect/proc_holder/spell/self/hive_dominance)
+			owner.AddSpell(new /obj/effect/proc_holder/spell/self/hive_dominance/dominance_spell)
```
Comment (Archanial) [inline code/modules/antagonists/hivemind/hivemind.dm]:
```suggestion
			owner.AddSpell(new /obj/effect/proc_holder/spell/self/hive_dominance)
```

**Review by Archanial — CHANGES_REQUESTED** [review]

**Review by Archanial — APPROVED** [review]

**Pull request merged**